### PR TITLE
Create onPRClose.yml

### LINKED ISSUE #3461: .github/workflows/onStagingCommits.yml → .github/workflows/onPRClose.yml
### Description

Change the name of the workflow to more accurately reflect its purpose.

### Goals

- [x] Change work flow filename
- [x] update github action

### Tasks
- [x] Change work flow filename
- [x] update github action

### PR DESCRIPTION
**Pull request by keifgwinn**
renaming onStagingCommits.yml

This fixes #3461 and assists #3460 

#### Description
- renamed the file
- added the new destroy action

#### Test cases
We can automatically close a branch on hosting

### CODE REVIEW

File: .github/workflows/onPRClose.yml [hunk 1]
```diff
@@ -21,3 +21,13 @@ jobs:
           issue-number: ${{ steps.commitMsgParser.outputs.issue-number }}
           label: 'in staging'
           github-token: ${{ secrets.GITHUB_TOKEN }}
+  destroy:
```
Comment (carmenfan) [inline .github/workflows/onPRClose.yml:24]: by the sound of the issue you only want to change the file name?

This block should belong in #3463 (if it's still applicable?)

**Review by carmenfan — CHANGES_REQUESTED** [review]

File: .github/workflows/onPRClose.yml (hunk 1 above)
Comment (keifgwinn) [reply .github/workflows/onPRClose.yml:24]: this is only a separate PR in order to get the github action to take the name change requested, it's the same piece of work as #3463. We can remove this block, merge this one, pull it back into #3463 and then make the change if you'd prefer

Comment (carmenfan) [reply .github/workflows/onPRClose.yml:24]: yes please

**Review by keifgwinn — COMMENTED** [review]
reverted destroy update

**Review by carmenfan — APPROVED** [review]

**Pull request merged**